src/odemis/driver/technolution.py

```diff
@@ -569,6 +569,23 @@ def _setScanDelay(self, scanDelay):
                          "%s" % (scanDelay[0], self.parent._mppc.acqDelay.value))
             return self.scanDelay.value
 
+    def _setResolution(self, field_resolution):
+        """
+        Sets the resolution so that the cell size (in both x and y direction) is a multiple of 4.
+
+        :param resolution (tuple):
```

Comment:
There is not much to add to these parameters isn't it?
It would just be repeating information which is already in the sentence:
Sets the resolution so that the effective cell size (in both x and y direction) is a multiple of 4.

Reply:
Repeating doesn't harm or even better rephrase ;) The input and return resolution are for example not necessarily the same value.
e.g. (feel free to make it even better)
:param resolution (tuple): requested resolution for a single field image. It is the product of the number of cell images multiplied by the effective cell size.
:return resolution (tuple): resolution closest possible to requested resolution for the single field image